api/tests/unit/experimentation/test_services.py

```
@@ -2505,3 +2506,186 @@ def test_annotate_warehouse_event_stats__clickhouse_connection__queries_customer
     # Then
     get_client.assert_called_once()
     assert getattr(fresh_connection, "event_stats", None) == expected_stats
+
+
+class TestGetExperimentFlagConfig:
+    def test__flag_disabled__returns_empty(
+        self,
+        organisation: "Organisation",
```

Ruff (0.16.2)

[error] Undefined name `Organisation`

(F821)